engines/registry/app/models/access_scope.rb

```diff
@@ -88,6 +88,21 @@ def allowed_paths(system = nil)
     allowed_products = (active_products & access_policies_yml.keys)
     allowed_glob_paths = access_policies_yml.values_at(*allowed_products).flatten
 
+    if !system.nil? && system.hybrid?
+      allowed_product_class = allowed_products.detect { |s| s.downcase.include?('ltss') }
+      unless allowed_product_class.empty?
+        activation_state = SccProxy.scc_check_subscription_expiration(
+          nil, system.login, system.system_token, Rails.logger, system.proxy_byos_mode, allowed_product_class
```

Comment:
if we need the headers for the `connect/systems/activations` endpoint (instead of `nil`), we would have to bring the headers all the way from the [top](https://github.com/SUSE/rmt/blob/master/engines/registry/app/controllers/registry/registry_controller.rb#L14), like the `client` to be able to access the system

Reply:
I sent an empty dictionary, instead of nil, as SccProxy.scc_check_subscription_expiration expects a dictionary here.

Reply:
The headers need to include an `Authorization` header with the system's credentials.

Reply:
addressed in #1209 in commit [9b1b344](https://github.com/SUSE/rmt/pull/1209/commits/9b1b344ad00468ed2bd355be3df725fbcd0435f4)